cumulusci/tasks/metadata/package.py

```diff
@@ -548,6 +548,12 @@ class UpdatePackageXml(BaseTask):
         "delete": {
             "description": "If True, generate a package.xml for use as a destructiveChanges.xml file for deleting metadata"
         },
+        "install_class": {
+            "description": "Specify post install class file to be used. Defaults to what is set in project config"
```

Comment:
@sf-jkunjoonju  These descriptions need to be parameterized

Reply:
@kdvenkata Can you clarify this request please? Parameterize how?